website/addons/s3/static/s3NodeConfig.js

```diff
@@ -91,31 +91,45 @@ ViewModel.prototype.updateBucketList = function(){
 
 ViewModel.prototype.selectBucket = function() {
     var self = this;
+
+    // The current version of boto does not handle bucket names containing periods
+    var isValidBucket = /^(?!.*(\.\.|-\.))[^.][a-z0-9\d-]{2,61}[^.]$/;
```

Comment:
It's defined but different. We still allow users to create buckets with periods in the names (because aws allows it) but we warn them that can't currently support it when they try to select it. They could also try to select a bucket containing periods they had created via the AWS console